src/runtime.rs

```diff
@@ -26,18 +28,55 @@ extern "system" {
     ) -> u32;
 }
 
-#[link(name = "windowsapp")]
-extern "system" {
-    pub fn CoIncrementMTAUsage(cookie: *mut RawPtr) -> ErrorCode;
+unsafe fn load_proc(library: &[u16], sym: &str) -> Result<RawPtr, ErrorCode> {
+    let library = LoadLibraryW(library.as_ptr());
+    if library.is_null() {
+        return Err(ErrorCode(0x8007_0000 | GetLastError()));
```

Comment:
Maybe wrap this in a `from_win32` function. Note that we tend to mask the result of `GetLastError` with `0x0000FFFF` when forming an `HRESULT`.